pkg/cmd/service/service_test.go

```diff
@@ -34,21 +34,27 @@ import (
 	healthv1pb "google.golang.org/grpc/health/grpc_health_v1"
 )
 
-func init() {
+var (
+	openFGAServerAddr string
+	openFGAGRPCAddr   string
+)
+
+func TestMain(m *testing.M) {
 	_, filename, _, _ := runtime.Caller(0)
 	dir := path.Join(path.Dir(filename), "../../..")
 	err := os.Chdir(dir)
 	if err != nil {
 		panic(err)
 	}
-}
 
-const (
-	openFGAServerAddr = "http://localhost:8080"
-	openFGAGRPCAddr   = "localhost:8081"
-)
+	config := DefaultConfig()
+	openFGAServerAddr = fmt.Sprintf("http://localhost%s", config.HTTP.Addr)
```

Comment:
`config.HTTP.Addr` is the addr. For example if I set `config.HTTP.Addr: "127.0.0.1:8082"` then the HTTP server will only bind on host address 127.0.0.1 and port 8082. 

So the `localhost` component here isn't quite right.

Reply:
🤔 but you're using the default config above which will be like `:8080`. This just feels weird. What's a better way to do it? I'm thinking..

Reply:
Use netip.AddrPort and make it explicit everywhere? I can do that in a following PR.

Reply:
I guess I'll need to do something like this with the changes I'm making for the random port. I'll include that in that PR.